src/utils/utils.js

```diff
@@ -769,6 +769,26 @@ const getStringId = (id) => {
     : JSON.stringify(id);
 };
 
+/**
+ * Function to return the state of consent management based on config passed in load options 
+ * @param {Object} cookieConsentOptions 
+ * @returns 
+ */
+const fetchCookieConsentState = (cookieConsentOptions) => {
+  let isEnabled = false;
+  // eslint-disable-next-line consistent-return
+  Object.keys(cookieConsentOptions).forEach((e) => {
+    if ( 
```

Comment:
it is a good practice on conditonals to assign logical results in a variable thatis selfdecraring of what the calculated value is before using it. Like:

const isSupportedAndEnabled = SUPPORTED_CONSENT_MANAGERS.includes(e) && 
      typeof(cookieConsentOptions[e].enabled) === 'boolean' && 
      cookieConsentOptions[e].enabled === true;

if(isSupportedAndEnabled) {

easier to read the code, split the value calculation if needed in separate util functions, reuse the result, amend the logic

Reply:
Adressed